### DISCUSSION TITLE
[Support]: False positive vs poorly framed objects as they pertain to Frigate+ labeling

### DISCUSSION THREAD

**Discussion by nickcmeyer** [blakeblackshear/frigate#15674] [General Support]
### Describe the problem you are having

What are the limits to false vs true positives for poorly framed, overlapping, and blurry objects? As you can see in the background there are multiple cars staggered. I'm wondering if labeling these cars (including the full extent of the car especially when partially hidden behind another car, marked as difficult) has resulted in more partial vehicles being detected even when the full vehicle is visible? Basically thinking that labeling these may have taught the model to offset a bounding box based on the foreground vehicle more than center a bounding box on the background vehicle even when the background vehicle is clearly identifiable by a human.

Now that I have these partial detections should I be marking them as false positives or true positive and re-framing? That likely includes relabeling all of the staggered vehicles in the back at the same time, again. Or going back and trying to delete the vehicles not in the foreground from past images? Kind of wish I could just mask out the far reaches when sending to frigate+ so that I don't have to worry about certain things polluting the model.

![Screenshot 2024-12-24 at 14-07-18 Plus - Frigate](https://github.com/user-attachments/assets/084dc2cb-fc4e-4426-b096-7793d0e1e9dd)

### Version

14

### Frigate config file

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Relevant go2rtc log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Install method

Docker Compose

### docker-compose file or Docker CLI command

```yaml
NA
```

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

NA

**Comment by NickM-27** [comment]
In this case it is fine to mark as true positive and redo the box. False positive is mainly used when there is no true object in the same area.

And it's important that all objects are labeled. I have some similar views on my panoramic wide angle camera and haven't had any issues like this.
